src/login-form.js

```diff
@@ -1,2 +1,34 @@
 
-export default undefined;
+export default class LoginForm {
+  constructor(form, email) {
+    this.userDirectory = [
+      { email: `aaron@theironyard.com`, password: `password123` },
+      { email: `admin@google.com`, password: `pandas` },
+      { email: email, password: `honeycrisp` },
+    ];
+  }
+
+  validate(username, password) {
+    return this.userDirectory.reduce((carry, current) => {
+      if (current.email === username && current.password === password) {
+        return true;
+      }
+
+      return carry;
+    }, false);
+  }
+
+  form.addEventListener(`submit`, (ev) => {
```

Comment:
This is causing a syntax error.

You are trying to run a statement in the middle of a class definition.
Statements need to be either part of a function or in raw areas of the code.

I think you may have meant to put this in the `constructor`.